rpm/oasis-server.spec

```diff
@@ -133,6 +133,9 @@ This package contains files for oasis-login.opensciencegrid.org
 
 
 %changelog
+* Tue Aug 16 2022 John Thiltges <jthiltges2@unl.edu> - 3.8-1
```

Comment:
The Version number also needs to be bumped, otherwise looks ok.

Reply:
Whoops. Thank you for catching that! Fix pushed.